src/content/pages/volunteers.mdx

```diff
@@ -13,16 +13,20 @@ We will also have a small number of perks for you, some small gifts to say "than
 
 # Things to remember
 
-* This is in person volunteering and would require you to be present at the venue.
+* This is **in-person** volunteering and would require you to be present at the venue.
 * While we strive to make the conference accessible to everyone, we cannot provide travel or accommodation support to volunteers.
-* To enter the conference you need a ticket.
-* You are welcome to volunteer as much or as little as you want to. We appreciate any help you can give us.
+* To enter the conference you need a ticket. Some volunteers are eligible for a free ticket, see perks below.
```

Comment:
```suggestion
* To enter the conference you need a ticket.
```
I do not want to have that much of a focus on this. And with this one it will be mentioned three times.